lib/config/migrations/custom/datasource-migration.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { DatasourceMigration } from './datasource-migration';
```

Comment:
PR titles should use the present tense, like this:

```diff
- refactor(datasource): renamed dotnet to `dotnet-version`
+ refactor(datasource): rename dotnet to `dotnet-version`
```

Reply:
Typo 😛